common/common_structs/src/farm_types.rs

```diff
@@ -81,8 +76,6 @@ pub trait FarmToken<M: ManagedTypeApi> {
     fn get_reward_per_share(&self) -> BigUint<M>;
 
     fn get_compounded_rewards(&self) -> BigUint<M>;
-
-    fn get_initial_farming_tokens(&self) -> BigUint<M>;
```

Comment:
Don't remove this method from the trait.

Reply:
Done.